docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -93,91 +102,114 @@ const session = new Session({ hostname: "jsonplaceholder.typicode.com"});
+
+    For example:
+    ```
+    children: [DirectoryContentsDefinition, TypicodeTodosDefinition]
+    ```
+    The command is defined in the list group. **[correct?]**
 
 ### Creating a command handler
 
-Also within the `typicode-todos` folder, create `TypicodeTodos.handler.ts`. Add the following code to the new file:
-```typescript
-import { ICommandHandler, IHandlerParameters, TextUtils, Session } from "@zowe/imperative";
-import { Typicode } from "../../../api/Typicode";
-export default class TypicodeTodosHandler implements ICommandHandler {
-
-    public static readonly TYPICODE_HOST = "jsonplaceholder.typicode.com";
-    public async process(params: IHandlerParameters): Promise<void> {
-
-        const session = new Session({ hostname: TypicodeTodosHandler.TYPICODE_HOST});
-        if (params.arguments.id) {
-            const todo = await Typicode.getTodo(session, params.arguments.id);
-            params.response.data.setObj(todo);
-            params.response.console.log(TextUtils.prettyJson(todo));
-        } else {
-            const todos = await Typicode.getTodos(session);
-            params.response.data.setObj(todos);
-            params.response.console.log(TextUtils.prettyJson(todos));
+1. In the `typicode-todos` folder, create the file `TypicodeTodos.handler.ts`.
+
+2. Add the following code to the `TypicodeTodos.handler.ts` file:
+    ```typescript
+    import { ICommandHandler, IHandlerParameters, TextUtils, Session } from "@zowe/imperative";
+    import { Typicode } from "../../../api/Typicode";
+    export default class TypicodeTodosHandler implements ICommandHandler {
+
+        public static readonly TYPICODE_HOST = "jsonplaceholder.typicode.com";
+        public async process(params: IHandlerParameters): Promise<void> {
+
+            const session = new Session({ hostname: TypicodeTodosHandler.TYPICODE_HOST});
+            if (params.arguments.id) {
+                const todo = await Typicode.getTodo(session, params.arguments.id);
+                params.response.data.setObj(todo);
+                params.response.console.log(TextUtils.prettyJson(todo));
+            } else {
+                const todos = await Typicode.getTodos(session);
+                params.response.data.setObj(todos);
+                params.response.console.log(TextUtils.prettyJson(todos));
+            }
         }
     }
-}
-```
-The `if` statement checks if a user provides an `--id` flag. If yes, we call `getTodo`. Otherwise, we call `getTodos`. If the
-Typicode API throws an error, the `@zowe/imperative` infrastructure will automatically surface this.
+    ```
+
+    The `if` statement checks if a user provides an `--id` flag. If yes, Zowe CLI **[correct?]** calls `getTodo`. Otherwise, Zowe CLI **[correct?]** calls `getTodos`. 
+    
+    If the Typicode API throws an error, the `@zowe/imperative` infrastructure automatically surfaces this. **[what is "this" in this sentence? What *thing* is surfaced? Why use the word "surface" here?]**
```

Comment:
@adam-wolfe please respond

Reply:
This sentence kind of comes out of nowhere, but I would change it to something like:
"If the Typicode API throws an error, the error will be forwarded to `@zowe/imperative` to log the error and print an error message to the terminal."

However, I'm not totally sure if this is a correct explanation. It would be good to have an engineer confirm what imperative does with thrown errors.